### PR TITLE
edited files in dir 0 and 1

### PR DESCRIPTION
**Pull request by FukurouMakoto**
I have edited files in dir 0 & 1. Please let me know if these are good and then I will continue work on the rest.

### CODE REVIEW

File: english/normal-dungeon/0/0-1A.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-Your muscle is 💪stronger 
+Your muscles have gotten stronger! 💪stronger
```
Comment (Arsenic-ATG) [inline english/normal-dungeon/0/0-1A.md:1]: Stronger repeated twice

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
Most of them look good to me. just a minor typo which must have been overlooked while editing.

File: english/normal-dungeon/0/0-1A.md (hunk 1 above)
Comment (FukurouMakoto) [reply english/normal-dungeon/0/0-1A.md:1]: Ah yes Totally right; don't know how I missed that. I'll make the edit now!

**Review by Arsenic-ATG — APPROVED** [review]

File: english/normal-dungeon/0/0-1A.md (hunk 1 above)
Comment (FukurouMakoto) [reply english/normal-dungeon/0/0-1A.md]: Change has been made!

**Review by ming-tsai — APPROVED** [review]
Hi @FukurouMakoto, thanks for you contributions and welcome to our team

**Pull request merged**